src/dra/843B0.c

```diff
@@ -1512,7 +1512,7 @@ void EntityHellfireBigFireball(Entity* entity) {
 
         entity->animSet = ANIMSET_DRA(9);
         entity->rotZ = 0;
-        entity->unk4C = &D_800B07C8;
+        entity->unk4C = &D_800B0798[12];
```

Comment:
Could you explain this change?

Reply:
I'm making an assumption that D_800B0798 is one contiguous `AnimationFrame` array since they all reference the same `ANIMSET_DRA(9)`

Reply:
@Xeeynamo is that how animsets work?

Reply:
I am suspicious of `&D_800B0798[12];` being real too for a simple reason. The previous word is:

```
/* 107C4 800B07C4 00000000 */ .word 0x00000000
```

this indicates the previous animation reached to an end. So `D_800B07C8` is a new animation. I do not think `&D_800B0798[12];` is valid in this context. If by a chance we are wrong, once we import the data the game will crash due to the index `D_800B0798` being out of bound. I prefer being wrong with a crash than being wrong and never knowing ;)

Reply:
Fixed in 23d66d78b501c4381cc7f5790cb06704e0b2868f!